src/ecommerce/seed/seed.js

```diff
@@ -0,0 +1,28 @@
+require('dotenv').config();
+const fs = require('fs');
+const { MeiliSearch } = require('meilisearch');
+
+const client = new MeiliSearch({
+  host: process.env.NEXT_PUBLIC_MEILI_HOST_NAME,
+  apiKey: process.env.NEXT_PUBLIC_MEILI_API_KEY,
+});
+
+const index = client.index('products');
+
+const data = require('./data.json');
+
+(async () => {
+  await index.updateFilterableAttributes([
+    'brand',
+    'category',
+    'tag',
+    'rating',
+    'reviews_count',
+    'price',
+  ]);
+  console.log('Filterable Attributes Updated');
```

Comment:
This is not accurate, the updateFilterableAttributes and the other update methods used below, just return a task uid which could fail. To confirm the settings have been updated, the status of the task should be checked.

Reply:
I have updated this, can you check @CaroFG